README.md

```diff
@@ -75,6 +75,37 @@ We recommend developers use Seq locally to help ensure the audit event developme
 
 ![Sample Seq Output](./assets/seq-events-view.png)
 
+## Using custom sink for large audit events
```

Comment:
Title of this chapter is wrong. We are not exposing a custom sink, but new `IAudit` implementation. I would call this chapter: _`SerilogLargeAuditEvents` implementation_